src/renderer/helpers/api/local.js

```diff
@@ -437,3 +437,33 @@ export function mapLocalFormat(format) {
     url: format.url
   }
 }
+
+/**
+ * video.js only supports MP4 DASH not WebM DASH
+ * so we filter out the WebM DASH formats
+ * @param {Format[]} formats
+ * @param {boolean} allowAv1 Use the AV1 formats if they are available
+ */
+export function filterFormats(formats, allowAv1 = false) {
+  const audioFormats = []
+  const h264Formats = []
+  const av1Formats = []
+
+  for (const format of formats) {
+    const mimeType = format.mime_type
+
+    if (mimeType.startsWith('audio/mp4')) {
+      audioFormats.push(format)
+    } else if (allowAv1 && mimeType.startsWith('video/mp4; codecs="av01')) {
+      av1Formats.push(format)
+    } else if (mimeType.startsWith('video/mp4; codecs="avc')) {
```

Comment:
How do we get these values? (In case there are new ones in the future)

Reply:
`console.log(formats)` then look at the `mime_type` property.